Tweets/_posts/What is a DAO/2021-06-30-What-is-a-DAO.md

```diff
@@ -1,117 +1,117 @@
 ---
 layout: post
-title:  "What is a DAO"
+title: "O que é um DAO"
 categories: [ Tweets ]
 image: ./What-is-a-DAO/1.jpg
 author: Tracheopteryx
-translator:
-publish: false
+translator: jameskbh
+publish: true
 ---
 
-original source: [twitter](https://twitter.com/tracheopteryx/status/1410243752434753547)
+fonte: [twitter](https://twitter.com/tracheopteryx/status/1410243752434753547)
 
 ##### 1
-Really enjoyed talking to [@TrustlessState](https://twitter.com/TrustlessState) on the [@BanklessHQ](https://twitter.com/BanklessHQ) podcast about [@coordinape](https://twitter.com/coordinape) with [@fifthworldzach](https://twitter.com/fifthworldzach)
+Gostei muito de conversar com [@TrustlessState](https://twitter.com/TrustlessState) no podcast [@BanklessHQ](https://twitter.com/BanklessHQ) sobre [@coordinape](https://twitter.com / coordinape) com [@fifthworldzach](https://twitter.com/fifthworldzach)
```

Comment:
`[@coordinape](https://twitter.com / coordinape)`

delete the space here that is why the build failed.

Reply:
changes made